p/i/index.php

```diff
@@ -27,8 +27,6 @@
 	require(APP_PATH . '/install.php');
 } else {
 	session_cache_limiter('');
-	Minz_Session::init('FreshRSS');
```

Comment:
Moved down to be called only once

Reply:
This prevented getting the name of the current user, thus preventing the HTTP cache from working.
Fixed in https://github.com/FreshRSS/FreshRSS/pull/6719